webpack.config.js

```diff
@@ -0,0 +1,92 @@
+// Webpack uses this to work with directories
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+// const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+// This is the main configuration object.
+// Here you write different options and tell Webpack what to do
+module.exports = {
+
+  // Path to your entry point. From this file Webpack will begin his work
+  entry: './src/javascript/index.js',
+
+  // Path and filename of your result bundle.
+  // Webpack will bundle all JavaScript into this file
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: './bundle.js',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      hash: false,
+      filename: './index.html',
+    }),
+    // new MiniCssExtractPlugin()
+  ],
+  module: {
+    rules: [
+      // {
+      //   test: /\.js$/,
+      //   exclude: /(node_modules)/,
+      //   use: {
+      //     loader: 'babel-loader',
+      //     options: {
+      //       presets: ['@babel/preset-env']
+      //     }
+      //   }
+      // },
```

Comment:
### Remove commented out code
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
Done.